core/packages/contracts/.gas-snapshot

```diff
@@ -1,29 +1,45 @@
-TokenVaultTest:testNonOwnerCannotDeposit() (gas: 20628)
-TokenVaultTest:testNonOwnerCannotWithdraw() (gas: 20672)
-TokenVaultTest:testTokenTransferFailedInsufficientAllowance() (gas: 70537)
-TokenVaultTest:testWithdrawSuccessful() (gas: 124729)
\ No newline at end of file
+BeefyClientTest:testSubmit() (gas: 646195)
+BeefyClientTest:testSubmitFailForPrevRandaoTooEarlyOrTooLate() (gas: 201121)
+BeefyClientTest:testSubmitFailWithInvalidBitfield() (gas: 588871)
+BeefyClientTest:testSubmitFailWithInvalidSignatureProof() (gas: 188116)
+BeefyClientTest:testSubmitFailWithStaleCommitment() (gas: 757402)
+BeefyClientTest:testSubmitFailWithoutPrevRandao() (gas: 566782)
+BeefyClientTest:testSubmitWithHandover() (gas: 674801)
+InboundQueueTest:testSubmit() (gas: 120028)
+InboundQueueTest:testSubmitShouldFailInsufficientBalance() (gas: 96221)
+InboundQueueTest:testSubmitShouldFailInvalidNonce() (gas: 67855)
+InboundQueueTest:testSubmitShouldFailInvalidProof() (gas: 65234)
+InboundQueueTest:testSubmitShouldNotFailOnHandlerFailure() (gas: 144077)
+InboundQueueTest:testSubmitShouldNotFailOnHandlerOOG() (gas: 643392)
+NativeTokensTest:testCreateFailOnBadFeePayment() (gas: 11388)
+NativeTokensTest:testCreateFailOnBadToken() (gas: 18421)
+NativeTokensTest:testCreateSuccessful() (gas: 75682)
+NativeTokensTest:testHandleRevertsUnknownOrigin() (gas: 11996)
+NativeTokensTest:testHandleRevertsUnknownSender() (gas: 45659)
+NativeTokensTest:testHandleUnlockMessageSuccessful() (gas: 151017)
+NativeTokensTest:testLockRevertsZeroAmount() (gas: 11366)
+NativeTokensTest:testLockSuccessful() (gas: 156773)
+OutboundQueueTest:testSubmit() (gas: 84448)
+OutboundQueueTest:testSubmitFailFeePaymentTooLow() (gas: 29771)
+ScaleCodecTest:testEncodeU128() (gas: 269)
+ScaleCodecTest:testEncodeU16() (gas: 245)
+ScaleCodecTest:testEncodeU256() (gas: 557)
+ScaleCodecTest:testEncodeU32() (gas: 247)
+ScaleCodecTest:testEncodeU64() (gas: 290)
+SolmateMerkleProofLibTest:testFuzzSample() (gas: 9528)
+SolmateMerkleProofLibTest:testFuzz_RandomProof(uint8,uint8) (runs: 1000, μ: 287794, ~: 340454)
+SolmateMerkleProofLibTest:testVerifyEmptyMerkleProofSuppliedLeafAndRootDifferent() (gas: 1512)
+SolmateMerkleProofLibTest:testVerifyEmptyMerkleProofSuppliedLeafAndRootSame() (gas: 1570)
+SolmateMerkleProofLibTest:testVerifyInvalidProofSupplied() (gas: 2203)
+SolmateMerkleProofLibTest:testVerifyLargeProofSupplied() (gas: 4479)
+SolmateMerkleProofLibTest:testVerifyShortProofSupplied() (gas: 1929)
+SolmateMerkleProofLibTest:testVerifyValidProofSupplied() (gas: 2227)
+TokenVaultTest:testDepositSuccessful() (gas: 108539)
+TokenVaultTest:testInsufficientBalance() (gas: 17294)
+TokenVaultTest:testNonOwnerCannotWithdraw() (gas: 48935)
+TokenVaultTest:testTokenTransferFailedInsufficientAllowance() (gas: 68190)
+TokenVaultTest:testWithdrawSuccessful() (gas: 124731)
+UpgradeProxyTest:testUpgrade() (gas: 30212)
+UpgradeProxyTest:testUpgradeFail() (gas: 20817)
+UpgradeProxyTest:testUpgradeFailBadOrigin() (gas: 10959)
```

Comment:
@doubledup after merging these changes, can you submit a PR to foundry to add a final newline for the output of this command?

Reply:
Noted this for after the new Solmate PR 👍